.drone.yml

```diff
@@ -7,3 +7,44 @@ pipeline:
     - /var/run/docker.sock:/var/run/docker.sock
     commands:
     - dapper ci
+
+  build-all-binaries:
+    privileged: true
+    image: rancher/dapper:1.11.2
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
+    environment:
+    - CROSS=1
+    commands:
+    - dapper build
+    when:
+      branch: master
+      event: tag
+   
+
+  stage-binaries:
+    image: rancher/dapper:1.11.2
+    commands:
+    - cp -r ./bin/* ./package/
+    when:
+      branch: master
+      event: tag
+
+  github_binary_release:
+    image: plugins/github-release
+    files: bin/*
+    secrets: [github_token]
+    when:
+      branch: master
+      event: tag
+   
+  publish-rke-certs-deployer:
```

Comment:
Regarding the other docker images, should they also be included as a part of publishing images to docker hub

Reply:
nginx proxy name: rancher/rke-nginx-proxy:0.1.0
rancher cert deployer: rancher/rke-cert-deployer:0.1.0
service side kick: rancher/rke-service-sidekick:0.1.0